iTechArt.Repository/Repositories/GraphRepository.cs

```diff
@@ -0,0 +1,113 @@
+﻿using iTechArt.Database.DbContexts;
+using iTechArt.Domain.Enums;
+using iTechArt.Domain.ModelInterfaces;
+using iTechArt.Domain.RepositoryInterfaces;
+using iTechArt.Repository.BusinessModels;
+using Microsoft.EntityFrameworkCore;
+
+namespace iTechArt.Repository.Repositories
+{
+    public sealed class GraphRepository : IGraphRepository
+    {
+
+        private readonly AppDbContext _dbContext;
+        public GraphRepository(AppDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        /// <summary>
+        /// Gets from database table-name for grocery and count of males and females in grocery table.
+        /// </summary>
+        public async Task<List<IGraph>> GetGroceryGraphData()
+        {
```

Comment:
why you are returning List of graphs? you always return collection with one element. In this case you don`t need any collections